### DISCUSSION TITLE
Tabs for repositories

### DISCUSSION THREAD

**Discussion by NN---** [community/community#2827] [Mobile]
There should be feature to work with multiple repositories.
It is easily done via browser but not using mobile application.

**Comment by candyho** [comment]
Hey thanks for sharing your feedback! Could you tell us a bit more about your workflow with multiple repositories? That'd be really help us to shape how this could work on mobile.

**Comment by NN---** [comment]
For instance I open 2 PRs in parallel or want to see a change in PR and then read documentation from other repository.
There are many use cases.

**Reply by candyho** [reply]
Awesome, that makes a lot of sense. I'll create an issue in our internal repo to keep track of this request and see what we can do here! Thank you for sharing more details with us.

**Reply by NN---** [reply]
Any update on this?
I do prefer using web browser instead of GitHub application just due to lack of this feature.
